src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker_Patterns.cs

```diff
@@ -127,6 +127,8 @@ public override BoundNode VisitITuplePattern(BoundITuplePattern node)
                 case BoundDeclarationPattern _:
                 case BoundDiscardPattern _:
                 case BoundITuplePattern _:
+                case BoundTypePattern _:
+                case BoundRelationalPattern _:
```

Comment:
>BoundRelationalPattern [](start = 21, length = 22)

seems like we should be able to learn from `and` patterns

Reply:
There is a PROTOTYPE comment on line 166 to that effect.

---
In reply to: [362991525](https://github.com/dotnet/roslyn/pull/40503#discussion_r362991525) [](ancestors = 362991525)